doc/source/building/windows.rst

```diff
@@ -8,250 +8,413 @@ Building From Source on Windows
+and for 32-bit run instead
+
+.. code:: shell
 
-We should be aware of the fact that these tools also install Python2, very similar to 
-a virtual environment, which is only usable within an MSYS2 terminal and we are **not**
-going to use it at any point. After updating, now we are going to use the build
-toolchain that we have installed in the previous step. Depending on 32/64bit choice,
-we will switch to another shell that MSYS2 created. In your start menu you should
-see three MSYS2 terminal shortcuts. Select the one with either 64 or 32bit indicator.
-The reason why we do this is that the toolchain and compilers are available to these
-shells and not to the standard MSYS2 terminal.
+    pacman -S --needed base-devel mingw-w64-i686-toolchain mingw-w64-i686-cmake git
 
-If you already have a GitHub repository folder where you keep your own repos, it is better 
-to use that location to keep things nice and tidy since we are going to clone yet another 
-repository to obtain the source code, hence
+Again, if you are not sure which one you want, choose 64-bit option in every
+step.
+
+It will prompt to whether install everything in these packages and you can
+simply accept all via hitting enter key at each step which also takes some time
+to complete. Once you install everything, just to have a fresh start, close and
+reopen the same terminal.
+
+If you already have a GitHub repository folder where you keep your own repos,
+it is better to use that location to keep things nice and tidy since we are
+going to clone yet another repository to obtain the source code, hence
 
 .. code:: shell
 
-    cd /c/<wherever the GitHub repo folder is>/GitHub
+    cd /c/<wherever you want to clone OpenBLAS into>/
 
-You don't necessarily need to build in that particular location, but it should be somewhere
-convenient. To make sure that we're ready to build, type the following in the terminal:
+It should be somewhere convenient and with write permissions. To make sure that
+we're ready to build, type the following in the terminal one-by-one:
 
 .. code:: shell
 
    make
    gfortran
    gcc
+   git
+
+Each of these commands should fail as we have not provided any arguments
+to them. However a failure also implies that they are accessible on the path
+which is what we wanted to test. In turn, if an error about the command being
+not found is returned then installation of the packages didn't complete
+succesfully. If any of these are missing, you're not ready to build. Go back
+and make sure that MSYS2 is installed correctly and has the required packages
+enabled.
 
-These commands should give errors as we have not provided any arguments to them.
-However an error also implies that they are accessible on the path. Now clone
-the repository required to build OpenBLAS:
+Now time to clone the OpenBLAS repository somewhere convenient; run the
+following line-by-line (the first line is added in case you didn't change
+directory yet otherwise you can ignore it).
```

Comment:
```suggestion
```
Let's keep a consistent policy of assuming the user follows the instructions correctly, adding extra detail only as needed to help with tricky steps or common pitfalls.

Reply:
Same as above